RELEASE/scripts/autoscend.ash

```diff
@@ -1,4 +1,4 @@
-since r20430;	//min mafia revision needed to run this script. Last update: Cargo Shorts support complete
+since r20494;	//min mafia revision needed to run this script. Last update: Cargo Shorts support complete
```

Comment:
```suggestion
since r20494;	//min mafia revision needed to run this script. Last update: retrocape skills
```
r20494 added retrocape skills support. which is why the since updated